simulators/predictive_simulator.py

```diff
@@ -102,7 +103,7 @@ def generate_innings(self, use_inferential_model):
 
         logging.debug("Initialising match state")
         match_state_dict = {}
-        self.simulated_matches_df.apply(lambda x: self.initialise_match_state(x, match_state_dict, playing_xi_df),
+        self.simulated_matches_df.swifter.apply(lambda x: self.initialise_match_state(x, match_state_dict, playing_xi_df),
```

Comment:
while it doesn't necessarily speed up the apply, it at least provides a helpful stat as it chugs along